tests/Paket.Tests/UpdateProcessSpecs.fs

```diff
@@ -348,27 +348,11 @@ let ``SelectiveUpdate conflicts when a dependency is contrained``() =
 
     (fun () ->
     selectiveUpdate true noSha1 (VersionsFromGraph graph) (PackageDetailsFromGraph graph) lockFile dependenciesFile
-        (PackageResolver.UpdateMode.UpdatePackage(Constants.MainDependencyGroup, PackageName "Castle.Core-log4net")) SemVerUpdateMode.NoRestriction
+        (PackageResolver.UpdateMode.UpdateFiltered(Constants.MainDependencyGroup, PackageFilter "Castle\\.Core-log4net")) SemVerUpdateMode.NoRestriction
     |> ignore)
     |> shouldFail
 
 [<Test>]
-let ``SelectiveUpdate does not update any package when package does not exist``() = 
```

Comment:
I removed this test as it's expectation didn't seem meaningful anymore. With pattern based updates, a pattern that didn't match any packages should be a no-op, not an exception.

The update package command still checks that the specific named package exists before calling UpdateFiltered